components/drawer/Drawer.razor.cs

```diff
@@ -502,37 +482,16 @@ private async Task HandleClose(bool isChangeByParamater = false)
             }
         }
 
-        private void CalcAnimation()
```

Comment:
I moved this to be inline with where it was called to make it obvious only the width or height transition could exist at once due to it being dependent on Placement's value. It is moved to line 490 below